TFT/src/User/Configuration.h

```diff
@@ -230,14 +230,25 @@
 #define ENABLE_UBL_VALUE 2
 
 /**
- * Enable friendly probe offset language.
+ * Enable friendly probe offset language
  * 
  * Decrease/increase and "-" & "+" icons are replaced with down/up and friendly icons
  * 
  * Options:  0: Disabled    1: Enabled
  */
 #define FRIENDLY_PROBE_OFFSET_LANGUAGE 1
 
+/**
+ * Enable quick EEPROM save/load/reset button
+ * 
+ * This will enable a handy button in the (settings > machine) menu
+ * 
+ * Note: if disabled, EEPROM operations can also be accessed in the (settings > machine > parameters) menu
+ * 
+ * Options:  0: Disabled    1: Enabled
+ */
+#define QUICK_EEPROM_BUTTON 0
```

Comment:
This should be left default ON. Although I will recommend using the EEPROM flag from M115 command to determine the visibility of this option.

Reply:
Whoops, this was supposed to be left on but I guess I left it at 0 during trsting